### ISSUE TITLE
Focus order is incorrect inside "Bitwise" pop up after "NOT" button.

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#961]
**Repro Steps**
1. Launch " Calculator " app.
2. Navigate to "Programmer" control in the navigation pane and activate it.
3. Navigate to 'Bitwise' button and press enter to activate it.
4. Navigate using tab key inside the pop up.

Observe the issue. 

**Actual Result**
Focus order is incorrect inside the "Bitwise" pop up after "NOT" button. After 
"NOT" button focus order is : "XOR" > "NAND" > "NOR"

**Expected Result**
Focus order should be correct inside the "Bitwise" pop up after "NOT" button as well. After "NOT" button, focus order should be : "NAND" > "NOR" > "XOR"

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.
